pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -1795,4 +1795,10 @@ type ScrapeClass struct {
 	// TLSConfig section for scrapes.
 	// +optional
 	TLSConfig *TLSConfig `json:"tlsConfig,omitempty"`
+
+	// ExtraRelabelings allow setting extra default relabelings to all
```

Comment:
It should be obvious that it applies to the target relabeling rules and when the scrape class' relabelings are applied compared to the monitor object's relabelings.